src/tests/ftest/util/collection_utils.py

```diff
@@ -476,7 +476,8 @@ def compress_files(logger, hosts, source, pattern, depth, test_result):
     """
     logger.debug("-" * 80)
     logger.debug("Compressing any %s files in %s on %s larger than 1M", pattern, source, hosts)
-    other = ["-size", "+1M", "-print0", "|", "sudo", "-n", "xargs", "-0", "-r0", "lbzip2", "-v"]
+    other = ("-size +1M -print0 "
+             "| xargs -0 -r0 -n1 -P $(nproc) sh -c 'sudo chmod go+r $0 && lbzip2 -v $0'")
```

Comment:
@phender This approach won't work. The files are potentially owned by another user. E.g. `daos_server`. And `lbzip2` tries to `chown daos_server <file>` after compressing. Which fails for the normal user
https://build.hpdd.intel.com/job/daos-stack/job/daos/job/PR-14532/1/artifact/Functional%20on%20EL%208.8/launch/functional_on_el_8.8/02-pool-create_all_vm-launch.log
```
2024/06/07 10:51:06 DEBUG                log_result_data:     lbzip2: compressing "/var/tmp/daos_testing/daos_server.log.39520" to "/var/tmp/daos_testing/daos_server.log.39520.bz2"
2024/06/07 10:51:06 DEBUG                log_result_data:     lbzip2: compressing "/var/tmp/daos_testing/daos_server.log.34839" to "/var/tmp/daos_testing/daos_server.log.34839.bz2"
2024/06/07 10:51:06 DEBUG                log_result_data:     lbzip2: fchown("/var/tmp/daos_testing/daos_server.log.39520.bz2"): Operation not permitted
2024/06/07 10:51:06 DEBUG                log_result_data:     lbzip2: "/var/tmp/daos_testing/daos_server.log.39520": compression ratio is 1:22.324, space savings is 95.52%
2024/06/07 10:51:06 DEBUG                log_result_data:     lbzip2: fchown("/var/tmp/daos_testing/daos_server.log.34839.bz2"): Operation not permitted
```

One approach might be:
```
sudo chown each file to normal user
lbzip2 as normal user
sudo chown back to original owner
```
But that seems tricky in the event of failures.

Reply:
Or maybe we could use a different compression tool and do
```
chmod go+r file
compress file to file.gz
rm file
```

Reply:
Or maybe we could just use `gzip` instead of `lbzip2`

`lbzip2` tries to chown the file:
```
[dbohning@boro-39 tmp]$ sudo touch file
[dbohning@boro-39 tmp]$ ll
total 0
-rw-r--r-- 1 root root 0 Jun 12 16:18 file
[dbohning@boro-39 tmp]$ lbzip2 file
lbzip2: fchown("file.bz2"): Operation not permitted
[dbohning@boro-39 tmp]$ ll
total 4
-rw------- 1 dbohning dbohninx 14 Jun 12 16:18 file.bz2
```

But `gzip` does not
```
[dbohning@boro-39 tmp]$ sudo touch file
[dbohning@boro-39 tmp]$ ll
total 0
-rw-r--r-- 1 root root 0 Jun 12 16:19 file
[dbohning@boro-39 tmp]$ gzip file
[dbohning@boro-39 tmp]$ ll
total 4
-rw-r--r-- 1 dbohning dbohninx 25 Jun 12 16:19 file.gz
```

Reply:
But `lbzip2` is multi-threaded so potentially faster. But maybe if we're going to use multi-process xargs it doesn't matter?

Reply:
Another idea is maybe instead of
```
chmod go+r file
lbzip2 file
```

We could do
```
chown $USER file
lbzip2 file
```

Reply:
Or, redirect `lbzip2` to `stdout` and a file manually
```
$ sudo touch file
$ lbzip2 file -c > file.bz2
$ ll
total 4
-rw-r--r-- 1 root     root      0 Jun 12 16:32 file
-rw-r--r-- 1 dbohning dbohninx 14 Jun 12 16:32 file.bz2
```